share/eds/utils.go

```diff
@@ -123,7 +123,7 @@ func CollectSharesByNamespace(
 
 	rootCIDs := ipld.FilterRootByNamespace(root, namespace)
 	if len(rootCIDs) == 0 {
-		return nil, nil
```

Comment:
We return non-nil everywhere else, correct?

Reply:
Correct. We create an empty slice on line 130 and then fulfil it further or return an empty slice

Reply:
I mean in everywhere in other getters

Reply:
found one more place. Thanks